azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -467,3 +552,73 @@ func resourceAzureRMKubernetesClusterServicePrincipalProfileHash(v interface{})
 
 	return hashcode.String(buf.String())
 }
+
+func base64Decode(str string) (string, bool) {
+	data, err := base64.StdEncoding.DecodeString(str)
+	if err != nil {
+		return "", true
+	}
+	return string(data), false
+}
+
+func getKubeConfig(config *string) *KubeConfig {
+	if config == nil {
+		return nil
+	}
+
+	configStr, error := base64Decode(*config)
+	if error == false && configStr != "" {
+		log.Println(config)
```

Comment:
given this is printing sensitive information, we should probably remove this?

Reply:
Woops, didn't mean to leave that line in!